packages/integration-karma/test/misc/cross-domain-iframe/index.spec.js

```diff
@@ -14,13 +14,14 @@ it('should not throw when unwrapping contentWindow', () => {
 });
 
 describe('HTMLIFrameElement.contentWindow patching', () => {
-    let iframe;
+    let iframe, sameOriginFrame;
 
     beforeAll(() => {
         const elm = createElement('x-test', { is: Test });
         document.body.appendChild(elm);
 
         iframe = elm.shadowRoot.querySelector('iframe');
+        sameOriginFrame = elm.shadowRoot.querySelector('#same_origin_iframe');
```

Comment:
```suggestion
        sameOriginFrame = elm.shadowRoot.querySelector('[id^="same_origin_iframe"]');
```

I totally forgot that we do some transformation with the ids. You can use what I suggested above or go by a css selector instead of id.